app_usb_aud_xk_316_mc/configs_test.cmake

```diff
@@ -13,14 +13,16 @@ set(APP_COMPILER_FLAGS_upgrade2 ${SW_USB_AUDIO_FLAGS} -DBCD_DEVICE_J=0x99
                                                       -DBCD_DEVICE_N=0x2)
 
 # Windows testing with the built-in driver relies on using product IDs that the Thesycon driver won't bind to
-set(APP_COMPILER_FLAGS_2AMi8o8xxxxxx_winbuiltin ${SW_USB_AUDIO_FLAGS} -DPID_AUDIO_2=0x001a)
+set(APP_COMPILER_FLAGS_2AMi8o8xxxxxx_winbuiltin ${SW_USB_AUDIO_FLAGS} -DPID_AUDIO_2=0x001a -DDFU_PID=0xd01a)
```

Comment:
Are these DFU_PID defines needed? I would expect them all to be set by xua_conf.h
Also for config_test.inc.